Update nadeESP.lua

Nexxed opened the pull request:
Changes:
- Added a color picker for changing the outline color
- Added a slider for changing the icon size
- Adjusted the offset of the icon so its more aligned to the center of the circle
- Moved the "enabled checks" out of the individual functions and to the main paint function instead (to reduce the number of `ui.get` calls)
- Removed semicolons since they're not needed in Lua
- Removed unused API functions from the `api` table to reduce file size

Nexxed committed:
Update nadeESP.lua

Changes:
- Added a color picker for changing the outline color
- Added a slider for changing the icon size
- Adjusted the offset of the icon so its more aligned to the center of the circle
- Moved the "enabled checks" out of the individual functions and to the main paint function instead (to reduce the number of `ui.get` calls)
- Removed semicolons since they're not needed in Lua
- Removed unused API functions from the `api` table to reduce file size

Velortex commented:
I have already implemented some of your suggestions and will work on the others.

Nexxed committed:
Update nadeESP.lua

Changes:
- Updated with the latest code from the repo
- Added fade-out over time checkbox for smoke and molotovs
- Cleaned-up code for better readability
- Removed API table since it actually further slows things down

 I also (yet again) moved the "selected" checks to the paint function again - it's better to have it there than cluttered in each draw function.

Nexxed commented:
FYI: you can merge this PR and have all of these changes applied to your file - these aren't suggestions, there are actual changes that I have made which you can see by clicking the "Commits" tab at the top...

Velortex commented:
Now all your changes should be applied
